be/src/runtime/routine_load/data_consumer.cpp

```diff
@@ -203,11 +204,14 @@ Status KafkaDataConsumer::group_consume(TimedBlockingQueue<RdKafka::Message*>* q
             }
             ++received_rows;
             break;
-        case RdKafka::ERR__TIMED_OUT:
+        case RdKafka::ERR__TIMED_OUT: {
             // leave the status as OK, because this may happened
             // if there is no data in kafka.
-            LOG(INFO) << "kafka consume timeout: " << _id;
+            std::stringstream ss;
+            ss << msg->errstr() << ", partition " << msg->partition() << " offset " << msg->offset();
+            LOG(INFO) << "kafka consume timeout: " << _id << " msg " << ss.str();
```

Comment:
How about logging config::routine_load_kafka_timeout_secon` here ?

Reply:
it's ok